esp-radio/src/wifi/mod.rs

```diff
@@ -57,7 +58,8 @@ pub(crate) use os_adapter::*;
 use portable_atomic::{AtomicUsize, Ordering};
 #[cfg(feature = "serde")]
 use serde::{Deserialize, Serialize};
-#[cfg(feature = "smoltcp")]
+#[cfg(all(feature = "smoltcp", feature = "unstable"))]
+#[cfg_attr(docsrs, doc(cfg(feature = "unstable")))]
```

Comment:
Shouldn't this be `#[cfg_attr(docsrs, doc(cfg(all(feature = "smoltcp", feature = "unstable"))))]`?